test/api/v3/unit/models/group.test.js

```diff
@@ -460,73 +460,67 @@ describe('Group Model', () => {
         };
       });
 
-      it('throws an error if no uuids or emails are passed in', (done) => {
+      it('throws an error if no uuids or emails are passed in', async () => {
```

Comment:
there's a problem with these tests, when you have an `async` function and you call something async like `Group.validateInvitations` you need to put `await` in front of it otherwise `Group.validateInvitations` will be called but not waited for and the function will continue to the next step regardless of what happens in `Group.validateInvitations`. So the tests are passing but actually the expec

Also this test was written to make sure that if `Group.validateInvitations ` didn't throw an error (the expected behavior) the test would timeout because `done()` wouldn't be called. Doing the same with `async/await` is less straightforward but you can try something like this

```
      it('throws an error if no uuids or emails are passed in', () => {
       let isSuccessful = false;
        try {
           await Group.validateInvitations(null, null, res);
           isSuccessful = true;
        } catch (err) {
            expect(err).to.be.an.instanceof(BadRequest);
            expect(res.t).to.be.calledOnce;
            expect(res.t).to.be.calledWith('canOnlyInviteEmailUuid');
        });

        if (isSuccessfull) throw new Error('Group.validateInvitations should have throw an error but it did not.')
   })
```